config/configauth/configauth.go

```diff
@@ -23,35 +23,25 @@ import (
 )
 
 var (
-	errAuthenticatorNotFound    = errors.New("authenticator not found")
-	errAuthenticatorNotProvided = errors.New("authenticator not provided")
+	errAuthenticatorNotFound = errors.New("authenticator not found")
 )
 
-// Authentication defines the auth settings for the receiver.
+// Authentication defines the auth settings for the receiver
```

Comment:
What happened to the final period?

Reply:
Fixed the period. It existed in the previous PR as well.